Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,81 @@
+var humanChoice = "";
+var humanScore = 0;
+var computerScore = 0;
+
+
+$('.choice').click(function() {
+   humanChoice = this.id;
+
+	 var choices = ["rock", "paper", "scissors"];
+
+	 var computerChoice = choices[Math.floor(Math.random() * choices.length)];
+
+   compare(humanChoice, computerChoice);
```

Comment:
Given that you're passing the `humanChoice` to the `compare` function as an argument, there's really no reason for `humanChoice` to be a global variable. Add the `var` on line 7 instead and get rid of line 1